docs/guide/snaps-exports.md

```diff
@@ -98,39 +98,56 @@ interface OnTransactionArgs {
 ### Returns
 
 ```typescript
+import { Component } from '@metamask/snaps-ui';
+
 type OnTransactionHandlerReturn = Promise<OnTransactionResponse>;
 
 interface OnTransactionResponse {
-  insights: { [key: string]: unknown };
+  content: Component;
 }
 ```
 
-- `onTransactionResponse` - The `insights` object returned by the snap will be displayed alongside the confirmation for the transaction that `onTransaction` was called with. Keys and values will be displayed in the order received, with each key rendered as a title and each value rendered as a string.
+- `onTransactionResponse` - The `content` object returned by the snap will be displayed using Custom UI alongside the confirmation for the transaction that `onTransaction` was called with. Keys and values will be displayed in the order received, with each key rendered as a title and each value rendered as a string.
```

Comment:
Custom UI is mentioned only at https://github.com/MetaMask/metamask-docs/blob/main/docs/guide/snaps.md#custom-ui-in-metamask-using-a-defined-set-of-components from what I can tell - mentioning it without defining it (and referring to the definition) may confuse the reader. Adding that section would be awesome, but otherwise maybe we can omit mentioning it for now?

Reply:
Ah, just noticed #618 :muscle:

Reply:
100% agree. But that link wouldn't be valid until the custom ui PR is merged.

Reply:
Resolved by doing one PR for both tasks